spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
```

Comment:
They can be wrapped in a method, but I the eager loading should be done here as is part of the specs code base, moving this to the rake task feels awkward to me. In the long run we should be able to have an option somewhere that just load all code on demand without this ugly hack.